src/Compilers/VisualBasic/Portable/Operations/VisualBasicOperationFactory.vb

```diff
@@ -1281,15 +1283,7 @@ Namespace Microsoft.CodeAnalysis.Operations
         Private Function CreateBoundBlockOperation(boundBlock As BoundBlock) As IBlockOperation
             Dim statements As Lazy(Of ImmutableArray(Of IOperation)) = New Lazy(Of ImmutableArray(Of IOperation))(
                 Function()
-                    ' We should not be filtering OperationKind.None statements.
```

Comment:
>OperationKind.None [](start = 49, length = 18)

I expected to see some tests with additional None operations in the printout, but I didn't see any. Is it that we don't have any way of producing None operation nowadays?

Reply:
@jcouv We do still produce none operations for NYI bound nodes. However, this change is just removing a piece of code that was dropping node operations that were direct children of bound block.

Reply:
We also have tests for these none operations. For example, you can search for `OperationKind.None` in the baselines for http://source.roslyn.io/#q=IOperationTests_IParameterReferenceExpression